submarine-sdk/setup.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setup(
+    name='pysubmarine',
+    version='0.1.dev',
```

Comment:
It would make more sense to keep the version string in sync with the Submarine version string itself. That said, I don't mind. And it is trickier to track/update all version strings in the project when we bump the version.

Reply:
change to 0.3.0-SNAPSHOT